gpigulevsky/CSS_02/CSS_03_03.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html> 
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Блоки</TITLE>
+		<style type="text/css">
+		body {
+			background-color: #ADAAAD;
+		}
+		.navi{
+			background-color:#C0C0C0; 
+			width: 15em;
+			float: left;
+		}
+		.art{
+			background-color: #808080;
+			max-width: 100%;
+			margin-bottom: 2em;
+			padding-left: 17em;
+			text-indent: 20px;
+		}
+		
+		.footer div {			
+			background-color:#C0C0C0;
+			clear: both;
+			text-align: center;
+		}
+		
+		img.icon{
+					width: 128px;
+			height: 128px;
+		}
+		img.align-right{
+			float: right;
+		}
+		.wmail {
+			padding-bottom: 140px;
```

Comment:
What is that? You need to "include" picture using another approach

Reply:
fixed